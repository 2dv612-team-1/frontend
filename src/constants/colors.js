export default {
  primary: '#3f51b5',
  secondary: '#e8eaf6',
};
