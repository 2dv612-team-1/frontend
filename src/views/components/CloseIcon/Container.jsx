import styled from "styled-components";

const Container = styled.div`
  position: relative;
  top: 0;
  right: 0;
  width: 20px;
  float: right;
  cursor: pointer;
`;

export default Container;
