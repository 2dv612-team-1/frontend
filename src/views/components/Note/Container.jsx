import styled from "styled-components";

const Container = styled.div`
  background: yellow;
  min-height: 500px;
  width: 500px;
  position: absolute;
  left: 50%;
  transform: translate(-50%, 0);
  top: 25%;
  padding: 20px;
`;

export default Container;
