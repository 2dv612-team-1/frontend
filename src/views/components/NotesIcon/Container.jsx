import styled from "styled-components";

const Container = styled.div`
  cursor: pointer;
  margin-top:5px;
  flex-basis: 50%;
`;

export default Container;
