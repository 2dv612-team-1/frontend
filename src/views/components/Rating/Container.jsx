import styled from "styled-components";

const Container = styled.div`
	cursor: pointer;
	float:left;
	margin-left: 250px;
`;

export default Container;
