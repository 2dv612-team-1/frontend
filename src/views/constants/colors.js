export default {
  primary: "#3f51b5",
  secondary: "#e8eaf6",
  gray: "#566172"
};
