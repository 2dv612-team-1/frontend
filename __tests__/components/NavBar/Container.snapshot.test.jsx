import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import "jest-styled-components";
import Container from "../../../src/components/NavBar/Container";
import colors from "../../../src/constants/colors";

describe("Container", () => {
  let props;
  let wrapper;

  const Wrapper = () => {
    if (!wrapper) {
      wrapper = shallow(<Container {...props}>{props.children}</Container>);
    }
    return wrapper;
  };

  // Reset Component and Props before each test.
  beforeEach(() => {
    props = {
      children: undefined
    };
    wrapper = undefined;
  });

  describe("when props are undefined", () => {
    it("Should match snapshot", () => {
      expect(toJson(Wrapper())).toMatchSnapshot();
    });

    it("should should use primary color", () => {
      const tree = Wrapper();
      expect(tree).toHaveStyleRule("background", colors.primary);
    });
  });
});
