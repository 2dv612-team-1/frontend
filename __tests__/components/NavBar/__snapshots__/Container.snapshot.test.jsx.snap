// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container when props are undefined Should match snapshot 1`] = `
.c0 {
  width: 100%;
  height: 50px;
  background: #3f51b5;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 20px 0 20px;
}

<div
  className="c0"
/>
`;
